### PR TITLE
Adds What-to-expect-from-a-design-sprint

### PR DESCRIPTION
**Pull request by atreni**

### CODE REVIEW

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+layout: post
```
Comment (poteto) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: You're missing the `---` at the top and bottom of this block of text!

Copy paste this:

```
---
layout: post
title: "What to expect from a Design Sprint"
social: true
author: Ashley Treni
twitter: "_ashleytreni"
github: atreni
summary: "participation, outcomes, and deliverables"
published: true
tags: design sprints, design strategy, UX design
---
```

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+tags: design sprints, design strategy, UX design
+
+A design sprint is a research strategy which focuses on and explores a feature or specific part of a product. With a limited scope and a short duration of a week, a sprint can test and validate ideas about user flow, functionality, and interaction design around a specific product goal. 
+
+At DockYard, we've been utlizing the design sprints methodolgy for the past several months, and refining what that process means for us. Design sprints begin with a planning phase, where we work with the client to develop a scope of work for each sprint duration. We've learned through experience what to expect about timing, activity, and production within a given sprint. However, for each new client, we must help them understand what to expect, which is quite different from a more traditional design process. Hands on involvement is an integral part of a design sprint, and it is imperative for us to set the expectations about participation up front. In the same vein, the outcomes and deliverables of a design sprint are low fidelity and more strategic in nature, so it is important to define the end goals and deliverables to support them.
```
Comment (poteto) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: typo: utilizing, methodology

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+At DockYard, we've been utilizing the design sprints methodology for the past several months, and refining what that process means for us. Design sprints begin with a planning phase, where we work with the client to develop a scope of work for each sprint duration. We've learned through experience what to expect about timing, activity, and production within a given sprint. However, for each new client, we must help them understand what to expect, which is quite different from a more traditional design process. Hands on involvement is an integral part of a design sprint, and it is imperative for us to set the expectations about participation up front. In the same vein, the outcomes and deliverables of a design sprint are low fidelity and more strategic in nature, so it is important to define the end goals and deliverables to support them. 
+
+To elaborate on Maria's blog post about Design Sprints (https://dockyard.com/blog/2015/08/17/design-sprints-what-are-they-for), here's a "what to expect" about participation, outcomes, and deliverables.
```
Comment (duggiefresh) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: Not a big deal, but you can hyperlink Maria's post by using the square brackets.

`[Maria's blog post about Design Sprints](https://dockyard.com/blog/2015/08/17/design-sprints-what-are-they-for)`

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+ * Product direction. For newer products, sprints can explore the wide landscape of possibilities to discover and prioritize the best ways to move forward. A sprint can help select and test directions early on, to determine the strongest product values to put into action.
+
+The deliverables for the following outcomes may come in various forms such as original files (Sketch, photoshop, illustrator documents), JPGs/PDFs/PNGs, links to online prototypes, and photographs.
+
+
+As you can see, the outcomes of a design sprint are more strategic in nature, and not a fully completed project. What you’ll take away from a sprint are research tools, the skeleton of an interface with core features and interactions, and most importantly, a better understanding of the next steps for your product. Setting these expectations for participation, outcomes, and deliverables up front is imperative to a successful design sprint. Transperency and communication are important tools for establishing trust and strong relationships with clients, and staying true to the project scope.
+
+We're excited to share more of our process with you, and will be launching a design sprints resource guide soon!
```
Comment (poteto) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: From a marketing angle, maybe we can leave a call to action here? Is there a mailing list, Twitter, etc they can follow to know when it's released?

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+ * Product direction. For newer products, sprints can explore the wide landscape of possibilities to discover and prioritize the best ways to move forward. A sprint can help select and test directions early on, to determine the strongest product values to put into action.
+
+The deliverables for the following outcomes may come in various forms such as original files (Sketch, photoshop, illustrator documents), JPGs/PDFs/PNGs, links to online prototypes, and photographs.
+
+
+As you can see, the outcomes of a design sprint are more strategic in nature, and not a fully completed project. What you’ll take away from a sprint are research tools, the skeleton of an interface with core features and interactions, and most importantly, a better understanding of the next steps for your product. Setting these expectations for participation, outcomes, and deliverables up front is imperative to a successful design sprint. Transperency and communication are important tools for establishing trust and strong relationships with clients, and staying true to the project scope.
```
Comment (rsocci) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: Typo: Transparency

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+tags: design sprints, design strategy, UX design
+---
+
+A design sprint is a research strategy which focuses on and explores a feature or specific part of a product. With a limited scope and a short duration of a week, a sprint can test and validate ideas about user flow, functionality, and interaction design around a specific product goal. 
+
+At DockYard, we've been utilizing the design sprints methodology for the past several months, and refining what that process means for us. Design sprints begin with a planning phase, where we work with the client to develop a scope of work for each sprint duration. We've learned through experience what to expect about timing, activity, and production within a given sprint. However, for each new client, we must help them understand what to expect, which is quite different from a more traditional design process. Hands on involvement is an integral part of a design sprint, and it is imperative for us to set the expectations about participation up front. In the same vein, the outcomes and deliverables of a design sprint are low fidelity and more strategic in nature, so it is important to define the end goals and deliverables to support them.
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I don't think we need this buildup paragraph

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+
+A stakeholder from the product in development is a very important participant in the design sprint, and they should expect to be hands on and present for the full sprint duration. Design sprints move very fast, and their industry knowledge and understanding of the product is key to a successful outcome. The team can move forward quickly and efficiently with the availability to ask questions and help make informed assumptions along the way. Stakeholders come with unique insights about the product that can illuminate conditions of the target market, knowledge about users, business goals, and any known limitations or pre-existing conditions up front.
+
+Planning the sprint objectives and project goals before the start of the sprint is important to stay focused on the task at hand. It can happen that other ideas may arise during a sprint - the design team leader will make sure to document for later, or redirect the sprint to address those considerations if they are integral to the solution.
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: "It can happen that other ideas may arise" is a long way to say "New ideas will come"

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+Part of establishing project goals is determining the outcomes and deliverables that result as a product of the sprint. The outcomes of a design sprint depend on the state of your product before the design sprint, and based on the project goals that were defined. The outcome of a sprint can look many different ways:
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I would say "...of a sprint can look very different:" or "...can go many different ways:"

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+ * A working prototype. Usually for a one-week design sprint, the result is a clickable prototype and one or two user stories. The thinking that went into defining the user flow and key interactions can be tested and validated through the prototype and user testing.
+
+ * Detailed wireframes and a full user flow. Often times, a series of sprints are held to explore a more complex product. With each week focusing on a different facet of the product, wireframes and design decisions can start to carry over to inform consecutive weeks. It is possible to have a developed user flow with several of the major screens sketched, prototyped, and even beginning to explore elements of visual design. With consecutive sprints, it is possible to explore the product more deeply and examine the connections and interdependencies between the proposed user flows.
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I think it should be clear that even if there are multiple consecutive weeks, testing will happen in each sprint

Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: Or maybe it's worth clarifying that up front and when we lead into these bullets we explain that the result may be a combination of these bullets

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+ * Preliminary visual design. If the content of a given sprint is narrow in scope, there may be time at the end of the sprint to begin visual design. Developing a visual style for the product can inform prototypes in consecutive sprints, or lay a framework for visual tone in the product itself.
+
+ * Product direction. For newer products, sprints can explore the wide landscape of possibilities to discover and prioritize the best ways to move forward. A sprint can help select and test directions early on, to determine the strongest product values to put into action.
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I'd remove the comma in the last sentence.

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+The deliverables for the following outcomes may come in various forms such as original files (Sketch, photoshop, illustrator documents), JPGs/PDFs/PNGs, links to online prototypes, and photographs.
+
+
+As you can see, the outcomes of a design sprint are more strategic in nature, and not a fully completed project. What you’ll take away from a sprint are research tools, the skeleton of an interface with core features and interactions, and most importantly, a better understanding of the next steps for your product. Setting these expectations for participation, outcomes, and deliverables up front is imperative to a successful design sprint. Transparency and communication are important tools for establishing trust and strong relationships with clients, and staying true to the project scope.
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I'd remove "as you can see,"

Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: and I'd also remove the comma after "nature" in the first sentence.

Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I would consider changing to: "...away from a sprint are research tools, the skeleton of an interface with core features and interactions and, most importantly, a better understanding..."

Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I don't think this line is necessary "Transparency and communication are important tools for establishing trust and strong relationships with clients, and staying true to the project scope."

File: source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+We're excited to share more of our process with you, and will be launching a design sprints resource guide soon!
```
Comment (kidfribble) [inline source/posts/2015-09-22-What-to-expect-from-a-design-sprint.md]: I would reword as "We're excited to share more of our process with you through our soon to be released (and free) design sprints resource guide! Signup to the DockYard newsletter below and we'll let you know when it has been released."
